docs/modules/policies/pages/authoring_tips.adoc

```diff
@@ -79,3 +79,44 @@ use({
     end,
 })
 ----
+
+==== JetBrains IDEs
+
+Go into the Preferences of JetBrains IDE of choice and navigate to the `JSON Schema Mappings` menu.
+
+----
+Preferences > Languages & Frameworks > Schemas and DTDs > JSON Schema Mappings
+----
+
+Add an entry with the following configuration
+
+[subs="attributes"]
+----
+Name: Cerbos
+Schema file or URL: {latest-schema-url}
+Schema version: JSON Schema Version 7
+File path pattern: cerbos/*
+----
+image:jetbrains-menu.png[alt="JetBrains JSON Schema Mappings menu",role="center-img"]
+
+NOTE: In the example above, the schema is applied to all files in the /cerbos directory.
```

Comment:
I am probably wrong here, but, Isn't the file path pattern relative in the above example? So presumably it applies to any files contained in a directory named cerbos?

```suggestion
NOTE: In the example above, the schema is applied to all files in the cerbos directory.
```

Reply:
Yes, you are right. My mistake.